packages/compass-import-export/src/import/import-csv.ts

```diff
@@ -165,13 +165,17 @@ export async function importCSV({
     await pipeline(...params);
   } catch (err: any) {
     if (err.code === 'ABORT_ERR') {
+      debug('importCSV:aborting');
```

Comment:
importCSV:aborting and importCSV:completing will log as soon as we abort or complete, before we write the import errors to file. importCSV:aborted and importCSV:completed should log with the result _after_ writing errors to file.

Hopefully that should be enough to show where things got to and where it went wrong the next time it flakes.